ui/js/components/perf/compare.js

```diff
@@ -108,3 +109,33 @@ treeherder.component('compareError', {
         newRevision: '<'
     }
 });
+
+treeherder.component('distributionGraph', {
+    template:`
+        <table class = "toolTipTable"> 
+            <tr> 
+                <td class = "valueColumn">{{$ctrl.minValue|reduceNumberLength}}</td> 
+                <td class = "distributionColumn"><canvas id="new" width="320" height="30"></canvas></td>  
+                <td class ="valueColumn">{{$ctrl.maxValue|reduceNumberLength}}</td> 
+            </tr>
+        </table>`,
+    bindings: {
+        replicates: '<'
+    },
+    controller: [ function () {
+        let ctrl = this;
+        let cvs = document.getElementById("new");
```

Comment:
It would probably be slightly more elegant to key off the id values off the name of the specific row the tooltip is associated with (e.g. `distribution-graph-row0-tooltip`), but I think this kind of hack is probably ok (since it's fairly isolated). Please rename the id to something more descriptive like `distribution-graph-new` and `distribution-graph-current` though -- just `new` and `current` is too generic (there would be a risk of different parts of the treeherder/perfherder code interfering with each other if made using used such names a habit).